src/main.rs

```diff
@@ -280,6 +280,7 @@ fn run() -> Result<()> {
     runner.execute(Step::Krew, "krew", || generic::run_krew_upgrade(run_type))?;
     runner.execute(Step::Gem, "gem", || generic::run_gem(&base_dirs, run_type))?;
     runner.execute(Step::Sheldon, "sheldon", || generic::run_sheldon(&ctx))?;
+    runner.execute(Step::Rtcl, "rtcl", || generic::run_rtcl(&ctx))?;
```

Comment:
Could you please tell me which tool is this about? I could not find it.

Reply:
`rtcl` is just **R tools for the command line**, and `rupdate` is one of its command, located in packages' `bin` or `win` subdir.

> `rupdate` to update all CRAN packages on your system, install missing CRAN packages listed in your collection `~/.rtcl/packages` and update all packages with a git source.

`jetpack` only update packages that installed from CRAN, `rtcl::rupdate` will update packages installed from CRAN and git resources (like GitHub, etc.).

The package is not on CRAN yet, and need to be installed from https://github.com/rdatsci/rtcl.